src/main/java/com/bobrust/gui/ApplicationWindow.java

```diff
@@ -259,16 +389,70 @@ public void mousePressed(MouseEvent e) {
 		versionPanel.add(aboutLabel);
 		return versionPanel;
 	}
-	
+
 	public Image getDrawImage() {
 		return drawImage;
 	}
-	
+
 	public Rectangle getCanvasRect() {
 		return canvasRect;
 	}
-	
+
 	public Rectangle getImageRect() {
 		return imageRect;
 	}
-}
+	private void loadButtonConfiguration() {
+		File configFile = new File(CONFIG_FILE_PATH);
+		if (configFile.exists()) {
+			try (FileReader reader = new FileReader(configFile)) {
+				config = gson.fromJson(reader, ButtonConfiguration.class);
+				LOGGER.info("Loaded button configuration from {}", CONFIG_FILE_PATH);
+
+				applyLoadedConfiguration();
+			} catch (IOException e) {
+				LOGGER.error("Failed to load button configuration", e);
+			}
+		} else {
+			LOGGER.info("Configuration file not found, creating new one with default values.");
+			saveButtonConfiguration();
+		}
+	}
+
+	private void applyLoadedConfiguration() {
+		if (config.color_topLeft != null && config.color_botRight != null) {
+			paletteRect.setBounds(
+					config.color_topLeft.x(),
+					config.color_topLeft.y(),
+					config.color_botRight.x() - config.color_topLeft.x(),
+					config.color_botRight.y() - config.color_topLeft.y()
+			);
+			LOGGER.info("Applied paletteRect from loaded configuration: {}", paletteRect);
+		}
+		screenDrawDialog.setPaletteRect(paletteRect);
+		Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
```

Comment:
This will not work as the screen size is dependent on the current monitor loaded